src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationServiceImpl registrationService
+            = new RegistrationServiceImpl();
+    private final StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+    private User actual;
+    private String login;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        actual = new User();
+    }
+
+    @Test
```

Comment:
Here and above plz rename your methods like here:
 For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>; For example, if we are testing the method register with a null
user's age the test method name should be register_nullAge_notOk. notOk is because the test expects the register method to throw an exception.

registrationValidUser_Ok() should be registration_ValidUser_Ok()